src/Util.re

```diff
@@ -0,0 +1,17 @@
+type inspectOptions;
+[@bs.obj]
+external makeInspectOptions:
```

Comment:
Does this need to be shared? What do you think about sharin after 3 usages?

Reply:
Hey sorry, I just saw this comment. This is actually a defined object in the Node.Util module: https://nodejs.org/api/util.html#util_util_inspect_object_showhidden_depth_colors

I don't know if this is exactly how we want to expose this options object, since ordinarily we would probably do the usual `@bs.obj` function with labelled params, but in this case, the options should be independently constructed, since it's used in a separate module here.

Reply:
I do think it's probably an odd placement for the options constructor, though. It's used by the `Inspector` type, and other functions that implicitly involve the inspector, but `Util` is a non-obvious place to find this constructor, from a Reason API standpoint.